app/editPlan/getTaskListData.js

```diff
@@ -0,0 +1,107 @@
+const NOT_STARTED = 'NOT STARTED'
+const CANNOT_START_YET = 'CANNOT START YET'
+const INCOMPLETE = 'INCOMPLETE'
+const COMPLETED = 'COMPLETED'
+
+const ADD_DETAILS = 'Add details'
+const REVIEW_DETAILS = 'Review details'
+
+const reviewComments = (comments, commentTypes) => {
```

Comment:
Would this be better as a helper function in common/utils, similar to getCommentText in there? Depends if we think it will be used much elsewhere I guess.

Reply:
I could expose it for now 🤔?
I think we could cover this in the validation spike though?

Reply:
Look at this as part of the validation spike (https://dsdmoj.atlassian.net/browse/SP-481).